runtime/databricks/automl_runtime/forecast/prophet/model.py

```diff
@@ -61,19 +64,35 @@ class ProphetModel(mlflow.pyfunc.PythonModel):
     """
     Prophet mlflow model wrapper for univariate forecasting.
     """
-    def __init__(self, model_json: Union[Dict[str, str], str], horizon: int, frequency: str) -> None:
+    def __init__(self, model_json: Union[Dict[str, str], str], horizon: int, frequency: str,
+                 time_col: str, target_col: str) -> None:
```

Comment:
I don't think `target_col` should be part of the API. It's more natural to do `y = model.predict(X)`, with the output returned rather than written back to `X` in place.

Reply:
The issue is that prophet model does not save the target column name and time column name. We have to do it in order to make the behavior the same with classification and regression.

Reply:
Can we get rid of the `target_col` parameter now? :)